pkg/api/login.go

```diff
@@ -37,11 +37,25 @@ func (hs *HTTPServer) ValidateRedirectTo(redirectTo string) error {
 	if to.IsAbs() {
 		return login.ErrAbsoluteRedirectTo
 	}
+
+	if to.Host != "" {
+		return login.ErrForbiddenRedirectTo
+	}
+
+	// path should . have exactly one leading slash
```

Comment:
Is it possible for double slashes to reach this point? Isn't it caught by `url.Parse`?
```suggestion
	// path should have exactly one leading slash
```